ResourceStatusSystem/PolicySystem/Actions/test/Test_RSS_PolicySystem_EmailAction.py

```diff
@@ -0,0 +1,84 @@
+''' Test_RSS_PolicySystem_EmailAction
+
+    this is pytest!
+'''
+
+import os
+import json
+from DIRAC.Core.Base.Script import parseCommandLine
+parseCommandLine()
```

Comment:
If you have troubles setting it up, you may probably need to work it out using "mock", and maybe "importlib"